jenkins/spark-tests.sh

```diff
@@ -78,11 +78,11 @@ getRevision() {
 set +x
 echo -e "\n==================== ARTIFACTS BUILD INFO ====================\n" >> "$tmp_info"
 echo "-------------------- rapids-4-spark BUILD INFO --------------------" >> "$tmp_info"
-p_ver=$(getRevision $RAPIDS_PLUGIN_JAR rapids4spark-version-info.properties)
+p_ver=$(getRevision $RAPIDS_PLUGIN_JAR rapids-4-spark_2.12-version-info.properties)
```

Comment:
use `SCALA_BINARY_VER` to tell if 2.12 or 2.13

Reply:
done.